pkgs/development/interpreters/conda/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchurl
+, runCommand
+, makeWrapper
+, buildFHSUserEnv
+, libselinux
+, xorg
+# Conda installs its packages and environments under this directory
+, installationPath ? "~/.conda"
+# Conda manages most pkgs itself, but expects a few to be on the system.
+, condaDeps ? [ stdenv.cc xorg.libSM xorg.libICE xorg.libXrender libselinux ]
+# Any extra nixpkgs you'd like available in the FHS env for Conda to use
+, extraPkgs ? [ ]
```

Comment:
This can be extended with an overlay as such:
```
$ cat ~/.config/nixpkgs/overlays/conda.nix                                                                                 ~/src/nixpkgs
self: super:
{
  conda = super.conda.override {
    extraPkgs = [ self.sssd self.firefox ];
  };
}

$ conda-shell
$ ls -l /usr/bin/firefox
lrwxrwxrwx 1 nobody nogroup 78 Dec 31  1969 /usr/bin/firefox -> /nix/store/a258dljnrj0chhnb6dpir2106yimfkb9-conda-shell-usr-target/bin/firefox

$ ls -l /usr/lib/sssd/libsss_krb5.so
lrwxrwxrwx 1 nobody nogroup 90 Dec 31  1969 /usr/lib/sssd/libsss_krb5.so -> /nix/store/a258dljnrj0chhnb6dpir2106yimfkb9-conda-shell-usr-target/lib/sssd/libsss_krb5.so
```